src/Tests/Execute/DeviceViewProvider.test.ts

```diff
@@ -114,28 +114,6 @@ suite("DeviceViewProvider", function () {
         done();
       });
     });
-    test("get Children under deviceManager Node", function (done) {
```

Comment:
This test case is related to `Executor` node type. `Executor` node type is not used anymore. So I remove this test case.